guides/common/modules/ref_registration-methods.adoc

```diff
@@ -0,0 +1,26 @@
+[id="registration-methods_{context}"]
+= Registration Methods
+
+There are the following main methods for registering hosts:
```

Comment:
This is a bit ambiguous to me. Maybe drop "main"? Or just say "there's global registration" and two deprecated methods?